com.overlayed.dev.yaml

```diff
@@ -0,0 +1,129 @@
+id: com.overlayed.Overlayed
```

Comment:
There is nothing in https://overlayed.com/ ? Did you read this link?

![image](https://github.com/user-attachments/assets/2ecc8d3d-57a0-40a7-883f-619d9a268e59)

Reply:
Marking this as blocked until this is resolved.

Reply:
This was a mistake in the copypasta i've corrected it now, keep in mind this PR is STILL a draft!